packages/integration-sdk-runtime/src/synchronization/index.ts

```diff
@@ -39,8 +39,37 @@ export enum RequestHeaders {
 export interface SynchronizeInput {
   logger: IntegrationLogger;
   apiClient: ApiClient;
-  integrationInstanceId: string;
+
+  /**
+   * The synchronization job `source` value.
+   *
+   * The JupiterOne bulk upload API requires a `source` value is provided to identify the source
+   * of the data being uploaded. When running in the managed infrastructure, `'integration-managed'` is used. When running
+   * outside the managed infrastructure, `'integration-managed'`, `'integration-external'`, or `'api'` can be used.
```

Comment:
I would like to see us disallow the use of `integration-managed` outside our managed infrastructure. I was quite surprised to realize we were always saying `integration-managed` when integrations are executed externally.